.github/workflows/zenodo-data-release.yml

```diff
@@ -55,23 +55,19 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: "us-west-2"
 
-      - name: Set up pudl-dev conda environment
-        uses: mamba-org/setup-micromamba@v2
+      - name: Install pudl environment using pixi
+        uses: prefix-dev/setup-pixi@v0.9.3
         with:
-          environment-file: environments/conda-lock.yml
-          environment-name: pudl-dev
-          cache-environment: true
-
-      - name: Install PUDL
-        run: |
-          pip install --no-deps --editable .
+          locked: true
+          cache: true
+          cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
 
       - name: Do a Zenodo data release
         env:
           ZENODO_SANDBOX_TOKEN_PUBLISH: ${{ secrets.ZENODO_SANDBOX_TOKEN_PUBLISH }}
           ZENODO_TOKEN_UPLOAD: ${{ secrets.ZENODO_TOKEN_UPLOAD }}
         run: |
-          zenodo_data_release \
```

Comment:
Miraculously this actually succeeded when I tested it. Maybe Zenodo is okay again?